vars/buildPlugin.groovy

```diff
@@ -89,6 +85,7 @@ def call(Map params = [:]) {
                                         '--update-snapshots',
                                         "-Dmaven.repo.local=$m2repo",
                                         '-Dmaven.test.failure.ignore',
+                                        '-Dfindbugs.failOnError=false'
```

Comment:
https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L577
```suggestion
                                        '-Dspotbugs.failOnError=false'
```

Reply:
What is the reason that one should change the flag in the `Jenkinsfile` and not in the pom? 
Here is an example of a properly configured pom: https://github.com/jenkinsci/analysis-pom-plugin/blob/master/pom.xml#L415

Reply:
I believe it's so that builds will fail locally but on CI the full build completes so as much information as possible is presented to the developer to fix in one iteration.

Reply:
Hmm, I think this is the wrong way. This is typically something for a Maven profile.